cmd/account/mgmt/account-assign.go

```diff
@@ -183,6 +195,100 @@ func (o *accountAssignOptions) tagAccount(accountIdInput string) error {
 	return nil
 }
 
+func (o *accountAssignOptions) buildAccount(seedVal int64) (string, error) {
+
+	fmt.Println("Creating account")
+	var newAccountId string
+
+	orgOutput, orgErr := o.createAccount(seedVal)
+	if orgErr != nil {
+
+		// If email already exists retry until a new email is generated
+		for orgErr == ErrEmailAlreadyExist {
+			orgOutput, orgErr = o.createAccount(seedVal)
```

Comment:
If we provide the same seedVal here, shouldn't it provide us with the same randomstring?